frontend/app/src/styled-components.ts

```diff
@@ -16,22 +16,29 @@
 
 import styled from "@emotion/styled"
 
-export const StyledApp = styled.div(({ theme }) => ({
-  position: "absolute",
-  background: theme.colors.bgColor,
-  color: theme.colors.bodyText,
-  top: theme.spacing.none,
-  left: theme.spacing.none,
-  right: theme.spacing.none,
-  bottom: theme.spacing.none,
-  overflow: "hidden",
-  "@media print": {
-    float: "none",
-    height: theme.sizes.full,
-    position: "static",
-    overflow: "visible",
-  },
-}))
+import { hasLightBackgroundColor } from "@streamlit/lib/src/theme"
```

Comment:
Shouldn't this be `from "@streamlit/lib"`?

Reply:
oh, you are right. Fixed it!